TMDbLib/Objects/Exceptions/NotFoundException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMDbLib.Objects.Exceptions
+{
+    public class NotFoundException : TMDbHttpException
+    {
+        public NotFoundException(HttpStatusCode httpStatusCode, TMDbStatusMessage statusMessage)
```

Comment:
In principle, this exceptionshould not know anything about any HttpStatusCode.

TMDbStatusMessage can stay - but we may encounter a scenario in which we don't have data for it. I'll try listing some exception types on the issue.